ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -239,6 +252,28 @@ StatusPopupMenu {
         }
     }
 
+    StatusMenuItem {
+
+         text: isVerificationRequestSent ||
+            root.incomingVerificationStatus === Constants.verificationStatus.verified ?
+            qsTr("ID Request Pending....") :
+            qsTr("Respond to ID Request...")
+        icon.name: "checkmark-circle"
+        enabled: root.isProfile && !root.isMe && root.isContact
+                                && !root.isBlockedContact && !root.isTrusted
+                                && (root.hasReceivedVerificationRequestFrom
+                                    || root.isVerificationRequestSent)
+        onTriggered: {
+            if (hasReceivedVerificationRequestFrom) {
+                root.openProfileClicked(root.selectedUserPublicKey, "respondToPendingRequest")
```

Comment:
can we make these states Enums instead? would be more efficient?

Reply:
Good idea. Done